Sources/App/Views/PublicPage.swift

```diff
@@ -90,10 +91,21 @@ class PublicPage {
                 .data(named: "turbolinks-track", value: "reload"),
                 .defer()
             ),
-            analyticsHead()
+            analyticsHead(),
+            postHead()
         )
     }
 
+    /// Any additional tags that should be inserted as the *first* tags in the `<head>`.
+    func preHead() -> Node<HTML.HeadContext> {
+        return .empty
+    }
+
+    /// Any additional tags that should be inserted as the *last* tags in the `<head>`.
+    func postHead() -> Node<HTML.HeadContext> {
+        return .empty
+    }
+
     /// if a search engine requests the page, we can tell it not to index it by including this meta tag.
```

Comment:
These ended up not being used, but they are useful overloads into the `head` so I'd like to keep them.